examples/shader/post_processing.rs

```diff
@@ -162,20 +78,254 @@ fn main_camera_cube_rotator_system(
+            // NOTE: positions are actually not used because the vertex shader maps UV and clip space.
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_POSITION,
+                vec![
+                    [-half_extents.x, -half_extents.y, 0.0],
+                    [half_extents.x * 3f32, -half_extents.y, 0.0],
+                    [-half_extents.x, half_extents.y * 3f32, 0.0],
+                ],
+            );
+            triangle_mesh.set_indices(Some(Indices::U32(vec![0, 1, 2])));
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_NORMAL,
+                vec![[0.0, 0.0, 1.0], [0.0, 0.0, 1.0], [0.0, 0.0, 1.0]],
+            );
+
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_UV_0,
+                vec![[2.0, 0.0], [0.0, 2.0], [0.0, 0.0]],
+            );
+            let triangle_handle = meshes.add(triangle_mesh);
+
+            // This material has the texture that has been rendered.
+            let material_handle = post_processing_materials.add(PostProcessingMaterial {
+                source_image: image_handle.clone(),
+                offset_r: Vec2::new(0.1f32, 0.1f32),
+                offset_g: Vec2::new(0.1f32, -0.1f32),
+                offset_b: Vec2::new(-0.1f32, -0.1f32),
+            });
+
+            commands
+                .entity(entity)
+                // add the handle to the camera so we can access it and change the percentages
+                .insert(material_handle.clone())
+                // also disable show_ui so UI elements don't get rendered twice
+                .insert(UiCameraConfig { show_ui: false });
+            if let Some(window_id) = option_window_id {
+                commands.entity(entity).insert(FitToWindowSize {
+                    image: image_handle.clone(),
+                    material: material_handle.clone(),
+                    window_id,
+                });
+            }
+            camera.target = RenderTarget::Image(image_handle);
+
+            // Post processing 2d fullscreen triangle, with material using the render texture done by the main camera, with a custom shader.
+            commands
+                .spawn_bundle(MaterialMesh2dBundle {
+                    mesh: triangle_handle.into(),
+                    material: material_handle,
+                    transform: Transform {
+                        translation: Vec3::new(0.0, 0.0, 1.5),
+                        ..Default::default()
+                    },
+                    ..Default::default()
+                })
+                .insert(post_processing_pass_layer);
+
+            // The post-processing pass camera.
+            commands
+                .spawn_bundle(Camera2dBundle {
+                    camera: Camera {
+                        // renders after the first main camera which has default value: 0.
+                        priority: camera.priority + 10,
```

Comment:
I changed that from `priority: 1` because it makes it easier to make it work with split screen example